src/main/resources/static/js/gameScreen.js

```diff
@@ -74,23 +72,22 @@ function handleLetterEntered(event){
     }
 
 
-
-
 }
-function createGrid(wordLength, totalGuesses){
+
+function createGrid(wordLength, totalGuesses) {
```

Comment:
Note to the group: this is likely a very intuitive interface... but avoid the urge to do in JavaScript what should be done in Spring Boot.  Remember the focus of this class is on services.